### DISCUSSION TITLE
Can Paulmann 501.34 be linked directly with lights or groups?

### DISCUSSION THREAD

**Discussion by okastl** [Koenkk/zigbee2mqtt#15549] [Devices]
Can Paulmann 501.34 be linked directly with lamps / groups via zigbee bindings?
The documentation https://www.zigbee2mqtt.io/devices/501.34.html doesn't say, that it can.

**Comment by okastl** [comment]
To answer my own question: Yes, it can. On/off and level works fine.
Attached is an updated device extension, adding 'brightness_move_up_1', 'brightness_move_down_1', 'brightness_move_stop_1','brightness_move_up_2', 'brightness_move_down_2', 'brightness_move_stop_2'

@Koenkk maybe you want to add it to herdsman converters.

[Paulmann.501.34.js.zip](https://github.com/Koenkk/zigbee2mqtt/files/10251859/Paulmann.501.34.js.zip)

**Reply by Koenkk** [reply]
Done!

**Reply by okastl** [reply]
> Done!

Great, I just tried 1.28.4-dev commit: [443eb23a](https://github.com/Koenkk/zigbee2mqtt/commit/443eb23a)
I believe you need to change

`    fromZigbee: [fz.command_on, fz.command_off, fz.battery],
`

to

`    fromZigbee: [fz.command_on, fz.command_off, fz.command_move, fz.command_stop, fz.battery],
`

otherwise it won't work.
